demo/admin/src/products/future/generated/ProductForm.tsx

```diff
@@ -255,7 +255,7 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
                             label={<FormattedMessage id="product.availableSince" defaultMessage="Available Since" />}
                         />
                         <Field name="image" isEqual={isEqual}>
-                            {createFinalFormBlock(PixelImageBlock)}
+                            {createFinalFormBlock(DamImageBlock)}
```

Comment:
@nsams should we close or merge #1904?

Reply:
close? we need both, those are two different issues